config.schema.json

```diff
@@ -1,182 +1,214 @@
+                          "type": "object",
+                          "condition": {
+                            "functionBody": "if (!model.local_api && model.sensors[arrayIndices] && model.sensors[arrayIndices].sensor_type && model.sensors[arrayIndices].sensor_type === 'Light Sensor') { return true; } else { return false; };"
+                          },
+                          "properties": {
+                              "value_key": {
+                                  "type": "string",
+                                  "enum": [
+                                      "brightness"
+                                  ]
+                              }
+                          }
+                      },
+                      "humidity_properties": {
+                          "title": "Humidity Properties",
+                          "type": "object",
+                          "condition": {
+                            "functionBody": "if (!model.local_api && model.sensors[arrayIndices] && model.sensors[arrayIndices].sensor_type && model.sensors[arrayIndices].sensor_type === 'Humidity Sensor') { return true; } else { return false; };"
+                          },
+                          "properties": {
+                              "value_key": {
+                                  "type": "string",
+                                  "enum": [
+                                      "relative_humidity"
+                                  ]
+                              }
+                          }
+                      },
+                      "temperature_properties": {
+                          "title": "Temperature Properties",
+                          "type": "object",
+                          "condition": {
+                            "functionBody": "if (!model.local_api && model.sensors[arrayIndices] && model.sensors[arrayIndices].sensor_type && model.sensors[arrayIndices].sensor_type === 'Temperature Sensor') { return true; } else { return false; };"
+                          },
+                          "properties": {
+                              "value_key": {
+                                  "type": "string",
+                                  "enum": [
+                                      "air_temperature",
+                                      "dew_point",
+                                      "feels_like",
+                                      "wind_chill"
+                                  ]
+                              }
+                          }
+                      },
+                      "motion_properties": {
+                          "title": "Motion Properties",
+                          "type": "object",
+                          "condition": {
+                            "functionBody": "if (model.sensors[arrayIndices] && model.sensors[arrayIndices].sensor_type && model.sensors[arrayIndices].sensor_type === 'Motion Sensor') { return true; } else { return false; };"
```

Comment:
Delete `!model.local_api && ` as `functionBody` applies to both Local and Cloud APIs.